.fixtures.yml

```diff
@@ -2,4 +2,6 @@ fixtures:
   repositories:
     concat:            "https://github.com/puppetlabs/puppetlabs-concat"
     extlib:            "https://github.com/voxpupuli/puppet-extlib"
+    foreman:           "https://github.com/theforeman/puppet-foreman"
+    redis:             "https://github.com/voxpupuli/puppet-redis"
```

Comment:
Do we need Redis? As long as we're not setting it up, we can leave the module out.

Reply:
Ah right, that's a good reason.